pkg/common/ingress/annotations/sessionaffinity/main.go

```diff
@@ -68,6 +74,13 @@ func CookieAffinityParse(ing *extensions.Ingress) *CookieConfig {
 		sn = defaultAffinityCookieName
 	}
 
+	ss, err := parser.GetStringAnnotation(annotationAffinityCookieStrategy, ing)
+
+	if err != nil || !affinityCookieStrategyRegex.MatchString(ss) {
+		glog.V(3).Infof("Invalid or no annotation value found in Ingress %v: %v. Setting it to default %v", ing.Name, annotationAffinityCookieStrategy, defaultAffinityCookieStrategy)
```

Comment:
This means an invalid configuration, so change the log level to `glog.Warningf`.

Reply:
I changed to a warning if there's an invalid strategy specified, but left as an info if there is no strategy specified since the intent was to have backwards compatibility and not require people to use this option. Otherwise, anyone who was using affinity but didn't set this would start getting warnings. Hope that's ok, but can change if necessary.